src/components/search/SearchResults.tsx

```diff
@@ -216,93 +242,104 @@ const SearchResults: FunctionComponent<SearchResultsProps> = ({ serverId = windo
+                        title={globalize.translate('Programs')}
+                        items={programs}
+                        cardOptions={{
+                            preferThumb: true,
+                            inheritThumb: false,
+                            showParentTitleOrTitle: true,
+                            showTitle: false,
+                            coverImage: true,
+                            overlayMoreButton: true,
+                            showAirTime: true,
+                            showAirDateTime: true,
+                            showChannelName: true
+                        }}
+                    />
+                    <SearchResultsRow
+                        title={globalize.translate('Channels')}
+                        items={channels}
+                        cardOptions={{ shape: 'square' }}
+                    />
+                    <SearchResultsRow
+                        title={globalize.translate('Playlists')}
+                        items={playlists}
+                    />
+                    <SearchResultsRow
+                        title={globalize.translate('Artists')}
+                        items={artists}
+                        cardOptions={{ coverImage: true }}
+                    />
+                    <SearchResultsRow
+                        title={globalize.translate('Albums')}
+                        items={albums}
+                        cardOptions={{ showParentTitle: true }}
+                    />
+                    <SearchResultsRow
+                        title={globalize.translate('Songs')}
+                        items={songs}
+                        cardOptions={{ showParentTitle: true }}
+                    />
+                    <SearchResultsRow
+                        title={globalize.translate('HeaderPhotoAlbums')}
+                        items={photoAlbums}
+                    />
+                    <SearchResultsRow
+                        title={globalize.translate('Photos')}
+                        items={photos}
+                    />
+                    <SearchResultsRow
+                        title={globalize.translate('HeaderAudioBooks')}
+                        items={audioBooks}
+                    />
+                    <SearchResultsRow
+                        title={globalize.translate('Books')}
+                        items={books}
+                    />
+                    <SearchResultsRow
+                        title={globalize.translate('Collections')}
+                        items={collections}
+                    />
+                    <SearchResultsRow
+                        title={globalize.translate('People')}
+                        items={people}
+                        cardOptions={{ coverImage: true }}
+                    />
+
+                    {allEmpty && !isLoading && (
+                        <div className='sorry-text'>{globalize.translate('SearchResultsEmpty', query)</div>
```

Comment:
I think I missed this in my previous suggestion :sweat_smile: 

```suggestion
                        <div className='sorry-text'>{globalize.translate('SearchResultsEmpty', query)}</div>
```